package.json

```diff
@@ -34,6 +25,7 @@
     "connect-thegraph": "yarn workspace @aragon/connect-thegraph",
     "connect-thegraph-voting": "yarn workspace @aragon/connect-thegraph-voting",
     "connect-thegraph-token-manager": "yarn workspace @aragon/connect-thegraph-token-manager",
+    "connect-types": "yarn workspace @aragon/connect-types",
```

Comment:
I think this is nice, but have we looked into possibly publishing individual types as per defined by [typescript](https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html), in a `@types/aragon-connect` package?

Reply:
We are already [publishing the types with our packages](https://github.com/aragon/connect/blob/master/packages/connect-core/package.json#L6), so I don’t think we need to publish on `@types/xyz`? Do you see a reason to do it?

The `@aragon/connect-types` package is meant to contain the types that are shared by our different packages, and its types to be exported by the different packages using them. We could also have them in `@aragon-core`, but having them in a separate package is something we discussed recently and wanted to try.

Reply:
Yeah, indeed there shouldn't be a need to publish them now; it's probably something we could do later if we ever need to. Wonder if having the types in a separate package could cause issues with linting tools? Guess that's why we're trying haha!

Reply:
> Wonder if having the types in a separate package could cause issues with linting tools? Guess that's why we're trying haha!

Things seem to work up to this point, but let’s keep an eye on it yes! :smile: